webapp/pages/index.vue

```diff
@@ -215,6 +230,10 @@ export default {
     openFilterMenu() {
       this.showFilter = !this.showFilter
     },
+    mouseLeaveFilterMenu() {
```

Comment:
I would like to have it in the code like in the dropdown menu, because it very often happens that we have to investigate the dom of the menu in the browser.
Every time we have to figure out how it works which always needs time. This way we can save a lot of this time.